beacon-chain/core/epoch/epoch_processing_test.go

```diff
@@ -751,9 +751,9 @@ func TestProcessSlashings_SlashedLess(t *testing.T) {
 				Balances:  []uint64{params.BeaconConfig().MaxEffectiveBalance, params.BeaconConfig().MaxEffectiveBalance},
 				Slashings: []uint64{0, 1e9},
 			},
-			// penality   = validator.effective_balance * min(sum(state.slashings) * 3, total_balance)   // total_balance
-			// 1500000000 = (32 * 1e9)                 * min(sum([0, 1e9])        * 3, ((32 * 1e9) * 2)) // ((32 * 1e9) * 2)
-			want: uint64(30500000000), // 32 * 1e9 - 1500000000
+			// penalty    = validator balance / increment * (3*total_penalties) / total_balance * increment
+			// 1000000000 = (32 * 1e9)        / (1 * 1e9) * (3*1e9)             / (64*1e9)      * (1 * 1e9)
```

Comment:
Please update these equations. For example, this still evaluates to 1500000000.0 in python

```python
>>> (32 * 1e9)        / (1 * 1e9) * (3*1e9)             / (64*1e9)      * (1 * 1e9)
1500000000.0
```

Reply:
while using uint